prow/README.md

```diff
@@ -163,7 +163,7 @@ Jenkins' URL, such as `http://pull-jenkins-master:8080`.
 6. Create a namespace for test pods named "test-pods".
 
  ```
- kubectl create -f cluster/test_pods_namespace.yaml
+ kubectl create -f cluster/test_pod_namespace.yaml
```

Comment:
This is configurable now so we should document that you need to make sure the namespace created here is what is set in the prow config. Also, instead of keeping an extra file, you could simply use `kubectl create namespace test-pod`.